exe/appmap

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'gli'
+
+require 'appmap/version'
+require 'appmap/hook'
```

Comment:
I see `AppMap::Config` being used here, but not `AppMap::Hook`.

Reply:
Required `appmap` instead both.